server/apps/users/forms.py

```diff
@@ -0,0 +1,61 @@
+from django import forms
+
+class UserProfileForm(forms.Form):
+
+    new_user = forms.BooleanField(label = 'Is this your first time accessing the site?', required=False)
```

Comment:
I guess this shouldn't necessarily be a question the user is asked, but rather handled internally where we know whether the account was just freshly created or not (e.g. by taking advantage of the django signals). 

Similarly, we can keep internally track whether a person has ever submitted the form/edited their profile responses to keep track of whether we should give a nudge to fill those.

Reply:
Thanks, yes, you're right, all of this functionality is due to be hidden from the user. At this stage this is just here to ensure there's a field available in the database table.